### PR TITLE
hub: add privileged to more hub jobs…

### PR DESCRIPTION
**Pull request by vdemeester**
# Changes

… all the ones that need docker-in-docker.

/cc @sthaha @afrittoli 

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/pipeline/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

File: tekton/images/test-runner/runner.sh [hunk 1]
```diff
@@ -54,7 +54,7 @@ if [[ "${DOCKER_IN_DOCKER_ENABLED}" == "true" ]]; then
     service docker start
     # the service can be started but the docker socket not ready, wait for ready
     WAIT_N=0
-    MAX_WAIT=5
+    MAX_WAIT=15
```
Comment (afrittoli) [inline tekton/images/test-runner/runner.sh:57]: ?

Comment (sthaha) [reply tekton/images/test-runner/runner.sh:57]: docker seems to fail to start some times in 5 seconds see: https://storage.googleapis.com/tekton-prow/pr-logs/pull/tektoncd_hub/11/pull-tekton-hub-unit-tests/1278590538689286145/build-log.txt

Comment (vdemeester) [reply tekton/images/test-runner/runner.sh:57]: @afrittoli it *is* just to give a little bit more time :stuck_out_tongue_closed_eyes:

**Pull request merged**